imap_processing/cdf/tests/test_cdf_attribute_manager.py

```diff
@@ -1,47 +1,121 @@
     cdf_manager = CdfAttributeManager(Path(__file__).parent.parent / "config")
+    # Test that default information was loaded in from "imap_default_global_cdf_attrs.yaml"
+    assert cdf_manager.global_attributes["Project"] == "STP>Solar-Terrestrial Physics"
+    assert cdf_manager.global_attributes["Source_name"] == "IMAP>Interstellar Mapping and Acceleration Probe"
+    assert cdf_manager.global_attributes["Discipline"] == "Solar Physics>Heliospheric Physics"
+    assert cdf_manager.global_attributes["Mission_group"] == "IMAP>Interstellar Mapping and Acceleration Probe"
+    assert cdf_manager.global_attributes["PI_name"] == "Dr. David J. McComas"
+    assert cdf_manager.global_attributes[
+               "PI_affiliation"] == "Princeton Plasma Physics Laboratory, 100 Stellarator Road, Princeton, NJ 08540"
+    assert cdf_manager.global_attributes["File_naming_convention"] == "source_descriptor_datatype_yyyyMMdd_vNNN"
 
-    cdf_manager.load_global_attributes("imap_default_global_cdf_attrs.yaml")
+    # Load in different data, test what was carried over
+    cdf_manager.load_global_attributes("imap_default_global_test_cdf_attrs.yaml")
+    assert cdf_manager.global_attributes["Project"] == "STP>Solar-Terrestrial Physics"
+    assert cdf_manager.global_attributes["Source_name"] == "IMAP>Interstellar Mapping and Acceleration Probe"
+    assert cdf_manager.global_attributes["Discipline"] == "Solar Physics>Heliospheric Physics"
+    assert cdf_manager.global_attributes["Mission_group"] == "Dysfunctional Cats"
+    assert cdf_manager.global_attributes["PI_name"] == "Ana Manica"
+    assert cdf_manager.global_attributes["PI_affiliation"] == "LASP, CU"
+    assert cdf_manager.global_attributes["File_naming_convention"] == "source_descriptor_datatype_yyyyMMdd_vNNN"
+    assert cdf_manager.global_attributes["Data_version"] == 2
+    assert cdf_manager.global_attributes["Test_attribute"] == "test"
 
     # Expected failure: file_naming_convention is not in schema.
     # for attr in cdf_manager.global_attributes.keys():
     #     assert attr in cdf_manager.global_attribute_schema.keys()
 
-    assert (
-        cdf_manager.global_attributes["Mission_group"]
-        == "IMAP>Interstellar Mapping and Acceleration Probe"
-    )
-
     # Load additional global attributes
-    cdf_manager.load_global_attributes("imap_mag_global_cdf_attrs.yaml")
+    cdf_manager.load_global_attributes("imap_test_global.yaml")
+
+    # Creating dictionary for loaded information
+    mag_test_global_attrs = cdf_manager.get_global_attributes("imap_test_T1_test")
+
+    assert mag_test_global_attrs["Mission_group"] == "Dysfunctional Cats"
+
+    # Testing if statement
+    assert mag_test_global_attrs["Descriptor"] == "TEST>Testinstrument"
+    assert mag_test_global_attrs["Logical_source"] == "imap_test_T1_test"
+    # Testing non required data from schema
+    assert mag_test_global_attrs["Data_type"] == "T1_test-one>Test-1 test one"
```

Comment:
Nice! I'd rename the variable away from "MAG" since it isn't using that attribute file anymore.